pkg/training/logs.go

```diff
@@ -78,6 +78,15 @@ func getInstanceName(job TrainingJob) (string, error) {
 	if len(pods) == 1 {
 		return pods[0].Name, nil
 	}
+	// if job is RayJob and the ray job pod name exists, return that name; otherwise, return the head pod name
+	if job.Trainer() == types.RayJob {
+		for _, pod := range pods {
+			if value, ok := pod.Labels["job-name"]; ok && value == job.Name() {
+				return pod.Name, nil
+			}
+		}
+		return job.ChiefPod().Name, nil
+	}
```

Comment:
As we discussed, arena will retrieve the latest chief pod by default, I think we do not need to add specific logic for ray job anymore.